### PR TITLE
Various typo fixes

### PR DESCRIPTION
**Pull request by leino**

### CODE REVIEW

File: chapters/VK_NV_win32_keyed_mutex/keyed_mutex_submit.txt [hunk 1]
```diff
@@ -5,14 +5,6 @@
...
 --
 
-When submitting work that operates on memory imported from a Direct3D 11
-resource to a queue, the keyed mutex mechanism may: be used in addition to
-Vulkan semaphores to synchronize the work.
-Keyed mutexes are a property of a properly created shareable Direct3D 11
-resource.
-They can: only be used if the imported resource was created with the
-etext:D3D11_RESOURCE_MISC_SHARED_KEYEDMUTEX flag.
-
```
Comment (r-potter) [inline chapters/VK_NV_win32_keyed_mutex/keyed_mutex_submit.txt:15]: This seems to differ from the other changes, in that it's not a simple typo tweak. What was the rational for removing these sentences?

Comment (leino) [reply chapters/VK_NV_win32_keyed_mutex/keyed_mutex_submit.txt]: I dropped the commit and filed an issue instead: https://github.com/KhronosGroup/Vulkan-Docs/issues/1435.

**Review by krOoze — APPROVED** [review]

File: chapters/pipelines.txt [hunk 2]
```diff
@@ -3477,7 +3477,7 @@ OpDecorate %2 SpecId 12 ; decorate our 32-bit floating-point constant
 ---------------------------------------------------
 
 From the above we have two specialization constants, one is a signed 32-bit
-integer and the second is a 32-bit floating-point.
+integer and the second is a 32-bit floating-point number.
```
Comment (oddhack) [inline chapters/pipelines.txt:3480]: We typically say "floating-point value".

Comment (leino) [reply chapters/pipelines.txt]: Fixed!

**Comment by leino** [comment]
Is there anything else that needs fixing? Can this be merged?

**Comment by oddhack** [comment]
> Is there anything else that needs fixing? Can this be merged?

It's good. We publish spec updates every week or two so PRs/MRs tend to get queued up and accepted in a batch. Thanks!

**Pull request merged**